Rakefile

```diff
@@ -1,13 +1,13 @@
-@files=[]
-
-task :default do
-  system("rake -T")
-end
-
-require "logstash/devutils/rake"
+require 'logstash/devutils/rake'
 
 desc "Compile and put filewatch jar into lib/jars"
 task :vendor do
-  exit(1) unless system './gradlew --no-daemon clean jar'
-  puts "-------------------> built filewatch jar via rake"
+  sh('./gradlew --no-daemon clean jar')
+end
+
+task :test do
+  require 'rspec'
+  require 'rspec/core/runner'
+  Rake::Task[:vendor].invoke
```

Comment:
Is there a reason we invoke `vendor` inside `test`, instead of depending on it?

```suggestion
task :test => :vendor do
  require 'rspec'
  require 'rspec/core/runner'
```

This seems pretty useful for _all_ of our plugins; should we define it in `devutils`?

Reply:
yeah, it's a bit strange but I thought there might have been a reason for not depending directly.
could be useful for devutils but we would need to have an empty `vendor` task as a default ...

Reply:
the idea for bringing it up here is that I've considered writing a Java test before in this plugin, 
so eventually having those would allow us to hook them up here and run them (at least manually).

to be side noted that the ".ci" conventions invoke `rspec` directly, instead of relying on a `rake test` to run *all* test, 
smt we would need to handle per plugin - having a default test task would help but lot of plugins need a review

Reply:
moved test task (base) definition to devutils: https://github.com/elastic/logstash-devutils/pull/96